test/FOPI_pions/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ function(run_one_energy
 endfunction()
 
 foreach(n RANGE 0 11)                  # loop over energy
-  float_math(E "0.4 + ${n}*0.1")  # calc. energy
+	float_math(E "round(0.4 + ${n}*0.1, 1)")  # calc. energy
```

Comment:
Please, align the indentation. Either 2 spaces as before or 4 spaces also for the line below (my preferred option, since in the other foreach above 4 spaces are used). It's just a detail, cmake dose not care about indentation as python, but, since we are here and it is a quick and easy fix, let's do things as best as possible. We include `clang-format `in the SMASH CI tests, so, while not essential, style matters...

Reply:
Thank you for for the hint, I adjusted the indentation.